docs/graphs/graph-sdk-example.md

```diff
@@ -0,0 +1,289 @@
+    - `groceryOrders`
+
+- Import test data from sample-data folder to created collections using console. Name of the files are the same as the collection names.
+
+## How to create the graph on Macrometa GDN
+Now that we have the data imported, we can create the graph on Macrometa GDN using javascript SDK.
+
+### Prerequisites
+
+- A free trial [Macrometa account](https://auth-play.macrometa.io/).
+- A Node.js 14+ installation on your local machine. You can download it from [here](https://nodejs.org/en/download/).
+- Install the Macrometa [javascript SDK](../sdks/install-sdks.md).
+
+:::note
+To check your Node.js version, run the following command in your terminal:
+
+```bash
+node -v
+```
+:::
+
+### Create a graph
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+
+```js
+const jsc8 = require("jsc8");
+
+client = new jsc8({
+  url: "https://play.paas.macrometa.io",
+  apiKey: "xxxxx",
+  fabricName: "_system",
+});
+
+// Please be sure that edge collections and vertices have been created before running this script.
+async function createGraph() {
+  const response = await client.createGraph("grocery-graph", {
+    edgeDefinitions: [
+      {
+        // Edge collection name --> This collection holds relationships between vertices
+        collection: "groceryOrders",
+        // Vertex collection that is used as the start vertex of the edge
+        from: ["groceryCustomers"],
+        // Vertex collection that is used as the start vertex of the edge
+        to: ["groceryItems"],
+      },
+    ],
+  });
+  console.log(response);
+}
+createGraph();
+```
+
+</TabItem>
+
+</Tabs>
+
+After we have created the graph, we can start querying it.
+
+##  How to query the graph
+
+### Query explanation
+Query for the recommendations for a customer can be done using the following query:
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="C8QL">
+
+
+```js
+
+// Query: Recommend the most popular item in the grocery store based on the number of users who purchased it
+LET doc = (FOR d in groceryItems FILTER d.name == @name RETURN d)[0]
+FOR user IN 1..1 INBOUND doc groceryOrders
+    FOR item IN 1..1 OUTBOUND user groceryOrders
+        COLLECT i=item.name WITH COUNT INTO c
+        FILTER i != @name
+        SORT c DESC
+        // LIMIT 2
+        RETURN {"item": i, "count": c}   // item -> number of users who purchased it
+                                         // count -> number of times it was purchased
+```
+
+</TabItem>
+
+</Tabs>
+
+The `@name` is a bind variable that can be set to any item name. In this example we can set it to `peanut butter`. First we are filtering by item name and then collecting all the users who purchased that item. Then we are collecting all the items that were purchased by those users. We are filtering out the item that we are looking (`peanut butter`) for and sorting the result by the number of times it was purchased.
```

Comment:
I'm confused. I thought this was an SDK example. Are we doing this query via SDK or in the web console?

Reply:
@diana-macrometa In SDK code, we are calling query worker, but I need to explain the query as well since that is the core of the example.